res/menu/gallery_drawer.xml

```diff
@@ -3,14 +3,26 @@
     xmlns:mupen64="http://schemas.android.com/apk/res-auto" >
 
     <item
+        android:id="@+id/menuItem_library"
+        android:icon="@drawable/ic_controller"
+        android:title="@string/categoryLibrary_title"/>
+
+    <item
         android:id="@+id/menuItem_settings"
         android:icon="@drawable/ic_settings"
-        android:title="@string/menuItem_settings">
+        android:title="@string/menuItem_settings"/>
+
+    <item
```

Comment:
Definitely.  I'm happy pushing this down as far as you want, even to the bottom TBH.